### PR TITLE
Canthv0 simpleadmin

### PR DESCRIPTION
**Pull request by Canthv0**
**Issue:**
Importing existing Get-SimpleAdminAuditlog script into github under a managed location.

**Reason:**
Moving the script to a proper location for maintaining.

### CODE REVIEW

**Comment by lusassl-msft** [comment]
It looks like this is failing due to formatting. 
Please run .build\CodeFormatter.ps1 -Save from PS7 to auto-format, and manually fix the issues called out by PSScriptAnalyzer.

**Review by bill-long — CHANGES_REQUESTED** [review]
The disclaimer should be removed. This is added by the build process to the released/signed script.

Also we generally do not declare authors in scripts, as these are intended to be collaborations. Anyone interested in determining authorship can examine the commit history.

**Comment by dpaulson45** [comment]
The readme should also be moved to under docs. 

@bill-long do we want this script under "Admin"?

**Comment by bill-long** [comment]
Admin sounds fine. I'll push a commit to move the readme.

File: Admin/Get-SimpleAuditLogReport.ps1 [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+		$ObjModified = ($ObjModified.split("\"))[-1]
+	}
+
+	# Get just the name of the cmdlet that was run
+	[string]$cmdlet = $_.CmdletName
+
+	# Build the result object to return our values
+	$Result | Add-Member -MemberType NoteProperty -Value $user -Name Caller
+	$Result | Add-Member -MemberType NoteProperty -Value $cmdlet -Name Cmdlet
+	$ResultResult | Add-Member -MemberType NoteProperty -Value $FullCommand -Name FullCommand
```
Comment (lusassl-msft) [inline Admin/Get-SimpleAuditLogReport.ps1:206]: Here is also a typo

**Comment by bill-long** [comment]
This PR is still pending action from @Canthv0 
* Remove disclaimer and author
* Fix typo identified by Lukas
* Run CodeFormatter -Save to fix the build issues as noted by Lukas

**Comment by dpaulson45** [comment]
@Canthv0 the script failed to match the formatting because of new requirement to have an empty line at the end of the file.

**Comment by Canthv0** [comment]
Ok I think I have everything fixed now.

**Pull request merged**